### DISCUSSION TITLE
Send table_name via query_params

### DISCUSSION THREAD

**Discussion by azhe403** [maplibre/martin#245] [Q&A]
Is possible send `table_name` via `query_params`?

```
curl \
  --data-urlencode 'table_name=public.table_name' \
  --data-urlencode 'objectParam={"answer" : 42}' \
  --get localhost:3000/rpc/public.function_source/0/0/0.pbf
```

**Comment by stepankuzmin** [comment]
Hi @azhe403 

Yes, that should be possible. In your case, `table_name` will be accessible as `query_params->'table_name'` in your `public.function_source` function. However, it may introduce security vulnerability, so be careful

**Reply by azhe403** [reply]
I've try it. and when creating function, i got error like in picture 2

![gambar](https://user-images.githubusercontent.com/17938038/133407337-fba91337-f9e4-47e6-b186-d7c0583559e4.png)
![gambar](https://user-images.githubusercontent.com/17938038/133407298-d99886c4-5438-4c47-a070-e973955921a2.png)

I've added `( )` replace `->` with `->>` but same.


Also I add declare variable too, but variable `table_name` known as name table, not a value of table_name

![gambar](https://user-images.githubusercontent.com/17938038/133410965-dd26409a-7f30-41d3-8516-99153942f83d.png)
![gambar](https://user-images.githubusercontent.com/17938038/133410755-52213b07-6f07-40c7-b883-465d7f5fbbba.png)

Any solution about this?
